doc/releases/v03.00.adoc

```diff
@@ -0,0 +1,56 @@
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.0.0[v3.0.0 - OSI "Cosmic Coulthard"]
+
+== Purpose
+
+This release is a complete rework including the feedback from many OSI users. Thank you!
+We are aligned with the ongoing discussions for the upcoming ISO 23150 standard for hardware sensor outputs in order to bring simulation and the hardware in the car closer together.
+
+https://github.com/OpenSimulationInterface/open-simulation-interface/pulls?page=1&q=is%3Apr+is%3Aclosed+milestone%3Av3.0.0[Content/Changes]
+
+* Added major version to package label (osi3), allowing concurrent use of OSI 2.x and OSI 3.x in one process.
+* Added new top-level messages (new concept SensorView (Configuration),etc.).
+** GroundTruth.
+** SensorView.
+** SensorData / FeatureData.
+** SensorViewConfiguration (used for auto-configuration of SensorView).
+* Added technology-specific SensorView sub-messages to aid physical models.
+* Introduced conceptual separation of “SensorModels” and “LogicModels”.
+* Extended enumerations (e.g. traffic sign Type according to StVO, vehicle Type, etc.).
+* Improved messages and definitions (additional RoadConditions, country code in GT, AmbientIllumination, etc.).
+* Use ISO 8855 coordinate systems (incompatible with earlier OSI versions).
+* Redefined DetectedXXX messages.
+* Introduced DetectedStationaryObjects.
+* Merged MovingObjects and Vehicle messages.
+* Redefined Lane messages (with separated lane boundary messages and intersections).
+* Redefined TrafficLight messages.
+* Redefined TrafficSigns (1 x MainSign + n x SupplementarySign inclusive BaseStationary).
+* New FeatureData messages (rework of previous LowLevelData message) - aligned with an upcoming standard ISO 23150.
+* FeatureData usable for fusion units.
+* Extension and clarification of the existing doxygen documentation.
+* Improved doxygen collaboration diagrams.
+* Build process improvements, added install targets.
+* Made proto definitions compatible with proto3 restrictions to allow concurrent use with proto3-only implementations in the future.
+* Fixed version-checking of transmitted data.
+* AND MUCH MORE ENJOY!!!
+
+https://github.com/OpenSimulationInterface/open-simulation-interface/files/2297162/osi_overview_v2.2_vs_v3.0.pdf>[See for more information the `changelog`]
+
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.0.1[v3.0.1 - OSI "Cosmic Coulthard", Update 1]
+
+https://www.youtube.com/watch?v=yG0oBPtyNb0[Summertime :)]
+
+== Purpose
+
+This release is a patch reworking parts of the documentation in order to avoid ambiguities and improving the build/installation process of OSI.
+
+https://github.com/OpenSimulationInterface/open-simulation-interface/pulls?page=1&q=is%3Apr+is%3Aclosed+milestone%3Av3.0.1[Content/Changes]
```

Comment:
Added missing header markup in context of PR #533 (9ac1d74d03234f5c351f4691c1accc995a9c33c8).